tests/test_core.py

```diff
@@ -8808,6 +8806,12 @@ def test_minimal_runtime_emscripten_get_exported_function(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_get_exported_function')
 
 
+  @node_pthreads
+  def test_binaryen_2170_emscripten_atomic_cas_u8(self, js_engines):
+    self.emcc_args += ['-s', 'USE_PTHREADS=1']
+    self.do_run_in_out_file_test('tests', 'binaryen_2170_emscripten_atomic_cas_u8', js_engines=js_engines)
```

Comment:
Does the test runner ever set that second argument (js_engines)?   I thought all tests just too a single argument (except the parameterized ones).

Reply:
@node_pthreads sets that argument. Without that pattern, the test is run without experimental SAB enabled in Node.js.